glustercli/cmd/volume-create.go

```diff
@@ -44,6 +44,10 @@ func init() {
 	volumeCreateCmd.Flags().IntVar(&flagCreateRedundancyCount, "redundancy", 0, "Redundancy Count")
 	volumeCreateCmd.Flags().StringVar(&flagCreateTransport, "transport", "tcp", "Transport")
 	volumeCreateCmd.Flags().BoolVar(&flagCreateForce, "force", false, "Force")
+	volumeCreateCmd.Flags().BoolVar(&flagReuseBricks, "reuse-bricks", false, "Reuse Bricks")
```

Comment:
nit: Doesn't have to be all caps. `Reuse bricks`, `Allow root directory`...and so on